monero-outreach-docs/translations/pt-br/storyboard/blockchain_gender_pt_br.md

```diff
@@ -0,0 +1,78 @@
+# Disparidade de gênero no blockchain: Problema ou potencial?
+
+Monica Mizzi | 12/09/18 Uma entrevista com @sunchakr da Monero Talk.

```

Comment:
Maybe we could use the male gender? Como se estivéssemos nos referindo ao programa "do Monero Talk"